pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# Overview
+
+*Provide a general overview of how your code contribution works, detailing the
+effects and consequences of the change.*
+
+# Current behaviour
+
+*Describe the current behaviour of the project and how it differs from the
+behaviour your change is proposing.*
+
+# Expected behaviour
+
+*Describe the behaviour your change is proposing to the project. If your change
+affects any compilation options or runtime flags, or changes expected
+input/output values, please document these effects here.*
+
+# Anything else we should know?
+
+* If there's any other relevant information we should know that may help us in understanding and verifying your patch, please include it here.
+
+# Checklist
+
+- [ ] I agree to follow the project [Code of Conduct](https://github.com/codeplaysoftware/oneapi-construction-kit/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
I'm not sure we need a checklist. Is it possible to just add these as a comment?

Reply:
Thanks